uicomponent-compose/feed/src/main/java/io/github/droidkaigi/feeder/feed/DroidKaigi2021ArticleItem.kt

```diff
@@ -0,0 +1,175 @@
+package io.github.droidkaigi.feeder.feed
+
+import androidx.compose.foundation.Image
+import androidx.compose.foundation.background
+import androidx.compose.foundation.clickable
+import androidx.compose.foundation.layout.aspectRatio
+import androidx.compose.foundation.layout.fillMaxWidth
+import androidx.compose.foundation.layout.padding
+import androidx.compose.foundation.layout.wrapContentWidth
+import androidx.compose.material.Icon
+import androidx.compose.material.IconToggleButton
+import androidx.compose.material.LocalContentAlpha
+import androidx.compose.material.MaterialTheme
+import androidx.compose.material.Text
+import androidx.compose.runtime.Composable
+import androidx.compose.runtime.CompositionLocalProvider
+import androidx.compose.ui.Modifier
+import androidx.compose.ui.graphics.Color
+import androidx.compose.ui.layout.ContentScale
+import androidx.compose.ui.res.painterResource
+import androidx.compose.ui.semantics.semantics
+import androidx.compose.ui.text.style.TextOverflow
+import androidx.compose.ui.tooling.preview.Preview
+import androidx.compose.ui.unit.dp
+import androidx.constraintlayout.compose.ConstraintLayout
+import androidx.constraintlayout.compose.Dimension
+import io.github.droidkaigi.feeder.FeedItem
+import io.github.droidkaigi.feeder.Media
+import io.github.droidkaigi.feeder.core.theme.ConferenceAppFeederTheme
+import io.github.droidkaigi.feeder.core.theme.typography
+import kotlinx.datetime.Instant
+import kotlinx.datetime.TimeZone
+import kotlinx.datetime.toLocalDateTime
+
+@Composable
+fun DroidKaigi2021ArticleItem(
+    favorited: Boolean,
+    onClick: () -> Unit,
+    onFavoriteChange: (FeedItem) -> Unit,
```

Comment:
We can remove favorite button, so we can remove `onFavoriteChange: (FeedItem) -> Unit,` and `favorited: Boolean,` 👍

Reply:
Removed favorite button: 7b98ad7

<img src="https://user-images.githubusercontent.com/29789286/135709973-fa48863e-cb25-4e1a-8c46-50c209d5c3ce.png" width="300px"/>